websites/A/AnizmTV/dist/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.7",
 	"author": {
-		"name": "Bas1C",
-		"id": "303603391941050370"
+		"name": "PuffyTR#0001",
+		"id": "219389887944589312"
```

Comment:
You cannot change the author if you didn’t make the presence.

Reply:
We got help from him on time, he also built another site. Since he already has ready-made codes, we asked him to adapt and add it to our site. At the moment, the computer is broken and can not get it due to financial situation. That's why he can't write himself.

Reply:
> do they have a new account?

You didn't answer my question

Reply:
Yes, he have.
Bashiku#6817
531417292446105620

Reply:
then add that one

Reply:
Added.